docs/oauth2.md

```diff
@@ -27,24 +27,33 @@ This [RabbitMQ authentication/authorisation backend](./access-control) plugin le
 This guide covers
 
  * [How it works](#how-it-works)
- * [Usage](#usage)
+    * [Prerequisites](#prerequisites)
+    * [Authorization Flow](#authorization-flow)
     * [Variables Configurable in rabbitmq.conf](#variables-configurable)
-    * [Resource Server ID and Scope Prefixes](#resource-server-id)
     * [Token validation](#token-validation)
+    * [Token Expiration and Refresh](#token-expiration)
     * [Scope-to-Permission Translation](#scope-translation)
     * [Topic Exchange scopes](#topic-exchange-scopes)
-    * [Using a different token field for the Scope](#use-different-token-field)
-    * [Using Tokens with Clients](#use-tokens-with-clients)
     * [Scope and Tags](#scope-and-tags)
-    * [Token Expiration and Refresh](#token-expiration)
+
+ * [Basic usage](#basic-usage)
+    * [Configure OAuth 2.0 provider's issuer](#configure-issuer)
+    * [Configure Signing Keys](#configure-signing-keys)
+    * [Using a different token field for the Scope](#use-different-token-field)
```

Comment:
Use gerund, titles finish in "ing" or non-gerund titles for section headings but keep them consistent as mcuh as possible so above I would change "Using" to "Use" to be consistent with "Configure"